source/MissionAction.cpp

```diff
@@ -186,11 +194,13 @@ void MissionAction::Load(const DataNode &node, const string &missionName)
 			conversation.Load(child);
 		else if(key == "conversation" && hasValue)
 			stockConversation = GameData::Conversations().Get(child.Token(1));
+		else if(key ==  "ship" && hasValue)
+			giftShips[GameData::Ships().Get(child.Token(1))] = (child.Size() > 2 ? child.Token(2) : "");
 		else if(key == "outfit" && hasValue)
 		{
 			int count = (child.Size() < 3 ? 1 : static_cast<int>(child.Value(2)));
 			if(count)
-				gifts[GameData::Outfits().Get(child.Token(1))] = count;
+				giftOutfits[GameData::Outfits().Get(child.Token(1))] = count;
```

Comment:
Should we print a warning when trying to give an outfit that doesn't exist in the GameData?

Reply:
Same as above, probably already handled by a generic check.